docs/manual/comparison_with_rdatatable.rst

```diff
@@ -0,0 +1,330 @@
+``DT[, .(m = x)]``                           ``df[:. {"m" : f.x}]``                      Rename and select column
+``DT[, .(sv=sum(v))]``                       ``df[:, {"sv": dt.sum(f.v)}]``              Sum column ``v`` and rename as ``sv``
+``DT[, .(v, v*2)]``                          ``df[:, [f.v, f.v*2]]``                     Return two columns, ``v`` and ``v`` doubled
+``DT[, 2]``                                  ``df[:, 1]``                                Select the second column
+``DT[, ncol(DT), with=FALSE]``               ``df[:, -1]``                               Select last column
+``DT[, .SD, .SDcols=x:y]``                   ``df[:, f["x":"y"]]``                       Select columns ``x`` through ``y``
+``DT[ , .SD, .SDcols = !x:y]``               | ``df[:, [name not in ("x","y")``          Exclude columns ``x`` and ``y``
+                                             |          ``for name in df.names]]`` or
+                                             | ``df[:, f[:].remove(f['x':'y'])]``
+``DT[ , .SD, .SDcols = patterns('^[xv]')]``  | ``df[:, [name.startswith(("x","v"))``     Select columns that start with x or v
+                                             |          ``for name in df.names]]``
+=========================================== ============================================ ==============================================
+
+In ``datatable``, a single column can also be selected as ``df['v']``.
+
+Subset rows and Select/Compute
+-------------------------------
+
+====================================             ==========================================          ==============================================
+RDatatable                                           Datatable                                              Action
+====================================             ==========================================          ==============================================
+``DT[2:3, .(sum(v))]``                            ``df[1:3, dt.sum(f.v)]``                             Sum column ``v`` over rows 2 and 3
+``DT[2:3, .(sv=sum(v))]``                         ``df[1:3, {"sv": dt.sum(f.v)}]``                     Same as above, new column name
+``DT[x=="b", .(sum(v*y))]``                       ``df[f.x=="b", dt.sum(f.v * f.y)]``                  Filter in ``i`` and aggregate in ``j``
+``DT[x=="b", sum(v*y)]``                          ``df[f.x=="b", dt.sum(f.v * f.y)][0,0]``             Same as above, return as scalar
+====================================             ==========================================          ==============================================
+
+In `R <https://www.r-project.org/about.html>`_, indexing starts at 1 and when slicing, the first and last items are included. However, in `Python <https://www.python.org/>`_, indexing starts at 0, and when slicing, only the first item is included.
+
+Grouping Operations - j and :func:`by()`
```

Comment:
I would call it simply "Grouping with by" just like we already do [here](https://datatable.readthedocs.io/en/latest/manual/groupby_examples.html).